datajoint/autopopulate.py

```diff
@@ -252,8 +256,7 @@ def handler(signum, frame):
         if reserve_jobs:
             signal.signal(signal.SIGTERM, old_handler)
 
-        if suppress_errors:
-            return error_list
+        return error_list, nkeys
```

Comment:
why does `populate` need to return `nkeys` now?

Reply:
This can be useful in automating workflows, since it allows us to determine how many keys were found to be processed.
For example, this features is used in the [neurophotonics](https://github.com/datajoint-company/neurophotonics) project in check_db. This allows the EC2 instance to continue working until the user stops inserting keys into the database, for those times that keys are inserted after the instance started, but before the initial workload is finished.

Reply:
Current users expect `populate` to return the error list. This change may break existing code. Changing the API should be done carefully.

Reply:
That is a valid concern, but I think it is also useful to be able to tell how many keys were processed.

I notice that there is nothing returned if `suppress_errors == False`, the default value. What if the return was left as just `error_list` in the case `suppress_errors == True`, and returned nkeys only if `suppress_erros == False`?